playbooks/ansible-test-splitter/run.yaml

```diff
@@ -1,8 +1,12 @@
 ---
 - hosts: all
   tasks:
+    - name: You are here
```

Comment:
Would you like to keep this in the merged version?

Reply:
Hehe, no thank you!